static/templates/admin_invites_list.hbs

```diff
@@ -7,9 +7,11 @@
         <span class="email">{{email}}</span>
         {{/if}}
     </td>
+    {{# if is_admin}}
```

Comment:
We spell this `{{#if ...` -- can you fix that?  Consistent whitespace is important for readability.

Reply:
ok done Tim